react-native-libraries.json

```diff
@@ -9611,9 +9609,15 @@
   {
     "githubUrl": "https://github.com/monkin77/react-native-esper-sdk",
     "nameOverride": "React Native Esper SDK",
-    "examples": [
-      "https://github.com/monkin77/react-native-esper-sdk/blob/master/example"
-    ],
+    "examples": ["https://github.com/monkin77/react-native-esper-sdk/blob/master/example"],
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/vonovak/react-native-theme-control",
+    "npmPkg": "@vonovak/react-native-theme-control",
+    "examples": ["https://github.com/vonovak/react-native-theme-control/tree/main/example"],
+    "ios": true,
+    "android": true,
+    "newArchitecture": true
```

Comment:
Hello @vonovak, thank you for adding new entry to the library! 🙏 

Just a small nit before we ship:
```suggestion
```

You package will be tagged automatically as using New Arch, since your packages.json includes `codegenConfig`. The manual field has been introduced only to handle edge-cases, when the autodetection fails.